package.json

```
@@ -39,6 +39,6 @@
     "rsvp": "^4.8.4"
   },
   "engines": {
-    "node": "6.* || 8.* || >= 10.*"
+    "node": "8.* || >= 10.*"
```

Node 11 is no longer supported by the node foundation, what do you think about using this instead?

```suggestion
    "node": "8.* || 10.* || >= 12.*"
```